start-local-development.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
add those scripts to package.json scripts like `"start-dev": "./start-local-development.sh"` so that people can do `yarn start-dev` / `yarn stop-dev`

Reply:
did some updates. was having issue running tests with ganache running on docker. don't know why so I changed docker-compose to start graph-node and start ganache manually